bridge/engine/EngineMixer.cpp

```diff
@@ -179,6 +164,61 @@ class RemovePacketCacheJob : public jobmanager::CountedJob
     size_t _endpointIdHash;
 };
 
+class FinalizeNonSsrcRewriteOutboundContext : public jobmanager::CountedJob
+{
+public:
+    FinalizeNonSsrcRewriteOutboundContext(bridge::EngineMixer& mixer,
+        transport::RtcTransport& transport,
+        bridge::SsrcOutboundContext& outboundContext,
+        bridge::EngineThreadContext& engineThreadContext,
+        bridge::EngineMessageListener& mixerManager,
+        uint32_t feedbackSsrc)
+        : CountedJob(transport.getJobCounter()),
+          _mixer(mixer),
+          _outboundContext(outboundContext),
+          _transport(transport),
+          _engineThreadContext(engineThreadContext),
+          _mixerManager(mixerManager),
+          _feedbackSsrc(feedbackSsrc)
+    {
+    }
+
+    void run() override
+    {
+        const size_t endpointIdHash = _transport.getEndpointIdHash();
+        const uint32_t ssrc = _outboundContext.ssrc;
+
+        auto packet = createGoodBye(ssrc, _outboundContext.allocator);
+        if (packet)
+        {
+            _transport.protectAndSend(std::move(packet));
+        }
+
+        if (_outboundContext.packetCache.isSet() && _outboundContext.packetCache.get())
+        {
+            // Run RemovePacketCacheJob inside of this job
+            RemovePacketCacheJob(_mixer, _transport, _outboundContext, _mixerManager).run();
+        }
+
+        _transport.removeSrtpLocalSsrc(ssrc);
+
+        _engineThreadContext.post(engine::bind(&EngineMixer::onOutboundContextFinalized,
+            &_mixer,
+            endpointIdHash,
+            ssrc,
+            _feedbackSsrc,
+            _outboundContext.rtpMap.isVideo()));
+    }
```

Comment:
This task finalize for non ssrc-rewrite outbound contexts. It must be called when the associated inbound stream is removed. Will do all this in a single Job:

- Send Rtcp Goodbye signalling that ssrc will be removed
- Delete video packet cache when outbound ctx keeps one
- Remove SrtpSsrc (does what old RemoveSrtpSsrcJob did)
- Call the EngineMixer on EngineThread to the context to be removed. This MUST be the last job run for this OuboundContext